docs/commands/doc.md

```
@@ -0,0 +1,45 @@
+  generate    Generate a brag document
+```
+
+### generate
+
+Generate a professional achievement document from your brags.
+
+You can filter which brags to include using IDs, categories, or tags.
+If no filters are specified, all brags will be included.
+
+Examples:
+  # Generate markdown document with all brags
```

markdownlint-cli2 (0.18.1)

Headings must start at the beginning of the line

(MD023, heading-start-left)